sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -120,6 +120,27 @@ static struct hdac_ext_stream *hda_get_hext_stream(struct snd_sof_dev *sdev,
 	return snd_soc_dai_get_dma_data(cpu_dai, substream);
 }
 
+static struct hdac_ext_stream *hda_ipc4_get_hext_stream(struct snd_sof_dev *sdev,
```

Comment:
Commit message. Maybe we can reword the end:

With this change, we can remove the HDaudio-specific setting in sof_ipc4_dai_config(), which will further simplify support for DMIC/SSP/SoundWire in LNL.